tests/test_units/test_dataio.py

```diff
@@ -55,14 +44,15 @@ def test_missing_or_wrong_config_exports_with_warning(monkeypatch, tmp_path, reg
 
     monkeypatch.chdir(tmp_path)
 
-    with pytest.warns(UserWarning, match=pydantic_warning()):
+    with pytest.warns(UserWarning, match="The global config"):
```

Comment:
Two UserWarnings are emitted everytime the config is invalid, this match string matches both of them.. hence the change.